### PR TITLE
Datepicker js optimization

### PR DESCRIPTION
**Pull request by tonmoy18**
The date picker ID prefix and number are in different vars
Pro: no need to read, slice and append to ID every time
Con: the original ID is changed.

### CODE REVIEW

**Comment by rotated8** [comment]
Hmmm. There are a bunch of extra files added to this PR.

File: app/assets/javascripts/datarepo/datepicker.js [hunk 1]
```diff
@@ -25,8 +23,7 @@ Blacklight.onLoad(function() {
     if ($(datepickerClass).length) {
         $.datepicker.setDefaults(datepickerOpts);
         $(datepickerClass).each(function() {
-            $(this).datepicker();
-
+	    $(this).datepicker();
```
Comment (rotated8) [inline app/assets/javascripts/datarepo/datepicker.js]: This got dedented a space. Please add it back.

**Comment by tonmoy18** [comment]
I think I should create a brand new branch, copy all the changes to that branch manually and then delete this branch. Please advise.

**Comment by rotated8** [comment]
I think this needs to be rebased against master. If you'd rather recreate the changes, that's fine, too.

**Comment by tonmoy18** [comment]
I had problems with this branch from the beginning as I did not recast in the first place, then after resolving merge conflicts I added and commuted again. Created a new branch, it has minimum changes

**Comment by rotated8** [comment]
:ok_hand:

**Pull request closed without merging**